src/schemas.ts

```diff
@@ -46,7 +46,7 @@ export const TOKEN_DATA_SCHEMA = {
     description: {
       type: 'string',
       minLength: 1,
-      maxLength: 150,
+      maxLength: 250,
```

Comment:
Could we go even higher?

Reply:
I don't see a reason not to. Increased to 1000.